roles/base_os/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+base_os_use_firewalld: "{{ False if ('openshift_master' in role_names or 'openshift_node' in role_names) else True }}"
```

Comment:
@twiest The easiest way would probably to remove the dependency on base_os (which is really at this point just making sure the firewall is turned on, installing bash_completion and setting the .vimrc and .irbrc for the root user. I would rather see a better way to apply those things on dev hosts without bringing them in for origin/enterprise installs. Any objections to moving those to a new role that is conditionally applied to hosts...  something like os_env_extras or something?

That said, my main idea with setting the default there is that I wasn't sure if the base_os role would have access to the variables that are set in the openshift_master and openshift_node roles, since the base_os role is processed before they are.